integration/map.go

```diff
@@ -32,6 +32,8 @@ import (
 
 const treeID = int64(0)
 
+var hasher = maphasher.Default
```

Comment:
To do it properly, yes. 

In the next PR, I'll be adding a `MapEnv` we can can do this with a table driven test.